quirks/password-rules.json

```diff
@@ -201,6 +201,9 @@
     "vanguard.com": {
         "password-rules": "minlength: 6; maxlength: 20; required: lower; required: upper; required: digit; required: digit;"
     },
+    "vetsfirstchoice.com": {
+        "password-rules": "minlength: 8; required: lower; required: upper; required: digit; allowed: [?!@$%^+=&]"
```

Comment:
Excellent writeup in your PR for this. I think I’ll use it as an example of a great PR!

Reply:
Can you toss a `;` at the end of the rule, just for stylistic sake? Then I’d be happy to merge.

Reply:
Certainly, done!